include/ck/utility/type_convert.hpp

```diff
@@ -393,7 +393,7 @@ inline __host__ __device__ bf8_t f8_convert_sr<bf8_t, half_t>(half_t x)
 #if defined(__gfx940__) || defined(__gfx941__) || defined(__gfx942__)
     // convert to float and use native converion
     return f8_convert_sr<f8_t>(type_convert<float>(x));
```

Comment:
@zjing14 could you also fix f8_t -> bf8_t here?

Reply:
f8->bf8_t?

Reply:
Yeah, it is probably a typo. The method converts half_t to bf8_t, nothing to do with f8_t.